src/silo/storage/reference_genome.cpp

```diff
@@ -0,0 +1,39 @@
+#include "silo/storage/reference_genome.h"
+
+#include <fstream>
+
+#include "silo/persistence/exception.h"
+
+namespace silo {
+
+ReferenceGenome::ReferenceGenome(std::vector<std::string> reference_genome)
+    : genome(std::move(reference_genome)) {}
+
+ReferenceGenome ReferenceGenome::readFromFile(const std::filesystem::path& reference_genome_path) {
+   if (!std::filesystem::exists(reference_genome_path)) {
+      throw std::filesystem::filesystem_error(
+         "Global reference genome file " + reference_genome_path.relative_path().string() +
+            " does not exist",
+         std::error_code()
+      );
+   }
+
+   std::ifstream reference_file(reference_genome_path.string());
+   std::vector<std::string> global_reference;
```

Comment:
Maybe rename to `reference_genomes` as it is no longer a globally shared variable